### PR TITLE
Proposed Fortify Replacement

### PR DESCRIPTION
**Pull request by RockyMichael**
## Describe your changes
-Added Aerobatics: Increases damage when in the air. Works with both standard melee attacks and damage from abilities. This passive ability is better suited for brute, as it enables more offensive play, as opposed to fortify, which prolonged fights and doesn't quite fit with the identity of the kit.
## Link to issue (if applicable)
N/A
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested my changes.

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/passives/Aerobatics.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    public Role getClassType() {
+        return Role.BRUTE;
+    }
+
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onDamage(CustomDamageEvent event) {
+        if (event.getDamager() instanceof Player damager) {
+            int level = getLevel(damager);
+            if (level > 0) {
+                if(UtilBlock.airFoliage(damager.getLocation().add(0,-1, 0).getBlock())){
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/passives/Aerobatics.java:57]: This also seems to apply while jumping.

Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/passives/Aerobatics.java:57]: Distance above ground should be in config

**Review by StudentAlleg — CHANGES_REQUESTED** [review]
There should be some way to tell that you are increasing damage with this skill.

Consider adding a cooldown.

I am not sure that this skill will work well, as in order to make is usable with seismic slam and takedown (the only two skills it makes sense with), it makes jumping and hitting very strong.

(not commenting on balance as suggested)

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/passives/Aerobatics.java (hunk 1 above)
Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/passives/Aerobatics.java:57]: should probobly be a isGrounded check instead, as the idea is that while you are airborne you deal more damage

Comment (RockyMichael) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/passives/Aerobatics.java:57]: This is intended, as the purpose of the ability is to jump to deal extra damage. The trade-off is that it is easier to be comboed while you jump, so you have to think tactically about when you want to jump for the extra damage.

**Comment by RockyMichael** [comment]
I agree that there should be a sound effect played to differentiate when the ability is used. Will add that.

**Comment by ezrn** [comment]
getConfig calls use the wrong names

**Comment by RockyMichael** [comment]
-Uses grounded check instead of checking 1 block below player for air. 
-Added sound to effect.
-Fixed config naming issues.

**Pull request closed without merging**